packages/fxa-content-server/app/scripts/models/auth_brokers/oauth-webchannel-v1.js

```diff
@@ -52,6 +52,14 @@ const OAuthWebChannelBroker = OAuthRedirectAuthenticationBroker.extend({
    * @private
    */
   onFxaStatus(response = {}) {
+    const skipCWTS =
+      response.capabilities && response.capabilities.skip_choose_what_to_sync;
```

Comment:
the other version of this could be `    const cwtsStatus = response.capabilities && response.capabilities.choose_what_to_sync || true;` , default to `true`?